start/data/osgconnect-storage.md

```diff
@@ -69,12 +70,13 @@ reached your `/home` quota, please contact us at
 
 ## `/public` Usage and Policies
 
-User directories within `/public` are meant **ONLY** for staging job files too large for 
-HTCondor file transfer (input greater than 100MB, output greater than 1GB). 
+User directories within `/public` are meant **ONLY** for staging job files too large for regular 
+HTCondor file transfer (per-job input greater than 100MB, or per-job output greater than 1GB), and should only use HTTP or StashCache 
+mechanisms (see tables for input and output, further below).
 
 **JOBS MUST NEVER BE SUBMITTED FROM WITHIN `/public`**, and should not list `/public` files in the 
```

Comment:
**JOBS MUST NEVER BE SUBMITTED FROM WITHIN `/public`**, Output should never be written to '/public' and should not list `/public` files in the